plugins/anchor-links/index.test.js

```diff
@@ -93,6 +100,87 @@ describe('anchor-links', () => {
       expect(headings).toMatchInlineSnapshot(`Array []`)
     })
 
+    test('adds correct tabbedSectionDepth for headings in <Tabs/>', () => {
```

Comment:
Added a test to specifically test that `tabbedSectionDepth` is working as expected.